docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -92,7 +92,8 @@ need to add a grant privilege to the MISO database from your remote machine:
 
 # Setting Up the Application Server
 
-Download the [Flyway command line tool](https://flywaydb.org/documentation/commandline/) and install it.
+Download the [Flyway command line tool](https://flywaydb.org/download/community) version 3.2.1 and install it.
```

Comment:
Later in the article we require Maven, and here we use the Flyway plugin for Maven, should we advise using that in the installation guide as well?

Reply:
We use the command-line (v3.2.1) tool when releasing to our staging and production servers to avoid needing the source code on them. I think recommending the same here is ideal in case users want to build and deploy on different machines